cypress/e2e/CMS/$InfographicCreation.feature

```diff
@@ -0,0 +1,286 @@
+        And user clicks on "Link" button to link to a media
+        And user clicks on "Select media" to choose a resource to link
+        And user enters "Automated Test Infographic" into media search box and clicks "Apply"
+        And user selects "Automated Test Infographic" item from media list
+        And browser waits
+        And user clicks on "Select media" button to select media item
+        And browser waits
+        And user selects "Published" from Save as dropdown
+        When user saves the content page
+
+
+    Scenario Outline: Verify infographic at the front end in the Article
+        Given user is navigating to the front end site with path site section plus "article-to-test-infographic"
+        And screen breakpoint is set to "desktop"
+        And page title is "Automated Test Article to test Infographic"
+        And the intro text reads "Article Intro Text"
+        And <descNumber> description reads "Infographic Display: Article: Large,Center:"
+        And image <imageNumber> is displayed at "<position>" with the source "<imageSource>" and alt text "<imageAlt>"
+        And the caption appears as "Automated Test Infographic Caption"
+        And the infographic link "View and Print Infographic" appears with href "infographic-desktop"
+
+        Examples:
+            | descNumber | imageNumber | position        | imageSource         | imageAlt |
+            | 1          | 1           | embedded-entity | infographic-desktop | Alt Text |
+            | 2          | 2           | align-left      | infographic-desktop | Alt Text |
+            | 3          | 3           | align-center    | infographic-desktop | Alt Text |
+            | 4          | 4           | align-right     | infographic-desktop | Alt Text |
+
+
+    Scenario: Verify infographic at the front end in the Home and Landing page
+        Given user is navigating to the front end site with path site section plus "home-and-landing-page-to-test-infographic"
+        # And the hero image is matching the earlier selected promo image of the hero image
+        And page title is "Automated Test Home and Landing page to test Infographic"
+        And multimedia card row has an infographic which matches the earlier selected promo image of Infographic
+        And multimedia card row displays the following cards
+            | title                             | link                                 |
+            | Automated Test Infographic        | {TEST_SITE_SECTION}/test-infographic |
+            | Article to test Related Resources | {TEST_SITE_SECTION}/article          |
+        And list card row title is "List Title"
+        And list row displays the following links
+            | title                             | link                                 | description                                          |
+            | Article to test Related Resources | {TEST_SITE_SECTION}/article          | Article to test Related Resources - Meta Description |
```

Comment:
since you will be removing Related recourses article from cms steps, get rid of it here too...

Reply:
Done